internal/generator/vector/sources.go

```diff
@@ -194,6 +206,47 @@ source = '''
 `
 }
 
+type SyslogReceiver struct {
+	ID            string
+	ListenAddress string
+	ListenPort    int32
+	Format        string
+}
+
+func (SyslogReceiver) Name() string {
+	return "syslogReceiver"
+}
+
+func (i SyslogReceiver) Template() string {
+	return `
+{{define "` + i.Name() + `" -}}
+[sources.{{.ID}}]
+type = "syslog"
+address = "{{.ListenAddress}}:{{.ListenPort}}"
+mode = "tcp"
+
+[sources.{{.ID}}.tls]
+enabled = false
+key_file = "/etc/collector/{{.ID}}/tls.key"
+crt_file = "/etc/collector/{{.ID}}/tls.crt"
+
+[transforms.{{.ID}}_split]
```

Comment:
these transforms are specific to k8s audit logs. I'd drop them first and then see what remaps if any are needed.

Reply:
Yes, I will be working precisely on this, I am ramping up with Vector's transform syntax